to_jaden_case.py

```diff
@@ -0,0 +1,6 @@
+"""Convert String to Jaden Case."""
+
+
+def to_jaden_case(string):
+    """Return every word in the input string in captialized form."""
+    return ' '.join(list(map(lambda x: x.capitalize(), string.split(' '))))
```

Comment:
Similarly to the sum from string kata, you can pass `str.capitalize` as the first argument to `map`, which is a little more concise. And as with one of the others, no need to cast a `map` result into a list. `str.join` can take any iterable!